source/FileCache.cs

```diff
@@ -44,7 +44,7 @@ static FileCache()
         /// </summary>
         public static CacheMode AppCacheMode { get; set; }
 
-        public static async Task<MemoryStream> HitAsync(string url)
+         public static async Task<MemoryStream> HitAsync(string url)
```

Comment:
minor nit-pick - fix the indentation to remove one extra space at the beginning of the line?